src/odemis/gui/cont/correlation.py

```diff
@@ -0,0 +1,247 @@
+
+# -*- coding: utf-8 -*-
+
+"""
+@author Patrick Cleeve
+
+Copyright © 2023, Delmic
+
+Handles the correlation .
+
+This file is part of Odemis.
+
+Odemis is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License version 2 as published by the Free
+Software Foundation.
+
+Odemis is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+Odemis. If not, see http://www.gnu.org/licenses/.
+
+"""
+import logging
+import numpy
+import wx
+# IMPORTANT: wx.html needs to be imported for the HTMLWindow defined in the XRC
+# file to be correctly identified. See: http://trac.wxwidgets.org/ticket/3626
+# This is not related to any particular wxPython version and is most likely permanent.
+import wx.html
+from odemis import model
+import odemis.acq.stream as acqstream
+import odemis.gui.model as guimod
+from odemis.acq.stream import StaticStream
+from odemis.gui.util import call_in_wx_main
```

Comment:
can you put the imports first, and then the from odemis import? See: https://github.com/delmic/odemis/blob/master/doc/develop/coding-guidelines.rst#import-order